tests/main/snap-info/task.yaml

```diff
@@ -27,3 +27,17 @@ execute: |
     snap info --verbose $TESTSLIB/snaps/basic-desktop
     snap info --verbose basic_1.0_all.snap|MATCH "sha3-384:"
     snap info --verbose test-snapd-tools|MATCH "  ignore-validation:"
+
+    echo "Ensure we show friendly error messages"
+    snap info no-such-snap > out.stdout 2> out.stderr
```

Comment:
The test will fail here, you need to add `|| :` or something like that to let the code continue.